src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -131,11 +131,15 @@ private string TryGetText(SyntaxToken token, SemanticModel semanticModel, Docume
             return string.Empty;
         }
 
-        private bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
+        private static bool TryGetTextForSpecialCharacters(SyntaxToken token, out string text)
```

Comment:
https://github.com/dotnet/roslyn/blob/71d5b9ed55854da44a74704a566a5b5828f0942f/.editorconfig#L278-L279

Looking weird this wasn't failing the build before?